_rules/element-lang-valid-de46e4.md

```diff
@@ -27,20 +27,21 @@ acknowledgments:
 
 This rules applies to any HTML element that:
 
-- has a [node document](https://dom.spec.whatwg.org/#concept-node-document) with a [content type](https://dom.spec.whatwg.org/#concept-document-content-type) of `text/html`; and
-- is a [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) of a `body` element; and
-- has a `lang` attribute that is not empty ("").
+- has a [node document][] with a [content type][] of `text/html`; and
+- is a [descendant][] in the [flat tree][] of a `body` element; and
+- has a `lang` attribute that is not empty (`""`).
 
 ## Expectation
 
 For each test target, the value of the `lang` attribute is a [valid language tag][].
 
 ## Assumptions
 
-- The `lang` attribute is assumed to be used to indicate the language of a section of the content. If the `lang` attribute is used for something else (for example to indicate a `code` element contains CSS), the content may still conform to WCAG despite failing this rule.
+- The language of the page can be set by other methods than the `lang` attribute, for example using HTTP headers or the `meta` element. These methods are not supported by all assistive technologies. This rule assumes that these other methods are insufficient to satisfying [Success Criterion 3.1.1: Language of Page][].
```

Comment:
This rule is about sections of content. I don't see how this is relevant.

Reply:
Agree with Wilco that this assumption is less relevant for this rule. Even if the page language is not set, a `lang` attribute on a page element would have relevance and would need to be valid. That `lang` attribute should indicate the language of everything within the element the attribute is on.